docs/src/modules/java/nav.adoc

```diff
@@ -4,6 +4,7 @@
 *** xref:event-sourced-entities.adoc[Event Sourced Entities]
 *** xref:key-value-entities.adoc[Key Value Entities]
 *** xref:http-endpoints.adoc[HTTP Endpoints]
+*** xref:grpc-endpoints.adoc[gRPC Endpoints]
```

Comment:
I was in doubt if this should only be one "Endpoints" with 2 subsections... I think if we would like to discuss the differences between them, that would be useful. On the other, I see them as 2 different components so like this made sense to me as well. All in one seemed clunky.

Reply:
I agree with the structural decisions you made here 👍

It also follows the previous convention established by the Entities docs. We have Event Sourced Entities and Key Value Entities in separate sections.

Putting HTTP and gRPC Endpoints all in one doc would be overwhelming and difficult to navigate.